src/hooks/useBreakpoints.ts

```diff
@@ -10,6 +18,8 @@ const BREAKPOINT_WIDTH: Record<Breakpoints, number> = {
   xl: 1280,
   "2xl": 1536,
   "3xl": 1920,
+  largedisplay: 2080,
+  fourK: 3000,
 };
```

Comment:
defined the type of largeddisplay and fourk breakpoint  and added the width of display